AprilSylph/XKit-Rewritten
# New Tweak: Stacked badges

Q: why
A: it was a more fun use of half an hour than trying to instrument an intentionally-delayed domcontentloaded event okay

### Description


Adds a tweak that stacks the "important internet checkmarks" unless you hover over them.

<img width="571" src="https://user-images.githubusercontent.com/8336245/217405795-daea420f-bf8c-4991-8685-5bf1401b6029.png">

~~Slightly out of date~~ hover animation demo:

<img width="571" src="https://user-images.githubusercontent.com/8336245/217402956-c880804a-a69a-4ecd-a75b-0ba884b8c07d.gif">

### Testing steps


- Confirm that badges are stacked, can be hovered, animate correctly, and still have popups when hovered.
- Confirm that stacked badges appear correctly in the regular and community header.
